lib/env-validator.ts

```
@@ -0,0 +1,116 @@
+    return process.env.NODE_ENV === 'production' || 
+           process.env.VERCEL_ENV === 'production'
+  }
+
+  /**
+   * Check if we're in development
+   */
+  static isDevelopment(): boolean {
+    return process.env.NODE_ENV === 'development' || 
+           process.env.VERCEL_ENV === 'development'
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)